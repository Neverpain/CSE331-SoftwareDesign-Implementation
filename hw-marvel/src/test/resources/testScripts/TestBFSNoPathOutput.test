# Tests the findPath method on two unrelated nodes and prints out
# the path from one node to the other

CreateGraph g

AddNode g n1
AddNode g n2

FindPath g n1 n2
