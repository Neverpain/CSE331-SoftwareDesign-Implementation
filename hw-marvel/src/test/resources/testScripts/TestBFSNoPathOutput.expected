# Tests the findPath method on two unrelated nodes and prints out
# the path from one node to the other

created graph g

added node n1 to g
added node n2 to g

path from n1 to n2:
no path found